### ISSUE TITLE
(Epoch 1) System freezes after lid close on laptop

### ISSUE THREAD

**Issue by mani2jeff** [pop-os/cosmic-epoch#566]
Love the work, UI and performance. Closing the lid on Acer Aspire laptop results in UI lock, but not suspend. Then opening the lid, the display is still on, but the UI is frozen/locked. Forced hard reset to return to operation. Looking forward to the continued work and improvements. Thanks

**Comment by maxed4901** [comment]
Hi. Do u have a swap partition assigned? Just asking cz after i created a swap partition on my last install of alpha, i noticed that issue didn't happen.  Yet!😁

**Comment by mani2jeff** [comment]
Yes, I confirmed the installer created a swap partition, but it appears to be memory based. Will create disk partition and see if that solves the issue.

**Comment by mani2jeff** [comment]
Applied updates this afternoon. No longer freezing after lid close. Swap space configured by installer working. Noticed that after return from suspend, the cursor not defaulting to the password field; have to click in the text box. That would be nice fix.

**Comment by jacobgkau** [comment]
Closing since it sounds like the suspend issue was fixed (and the password field is also focused by default these days).
